### PR TITLE
Add jongburmpark sydney file to project

### LINKED ISSUE #2: GitHub Teacher tracks steps using GitHub Flow
### GitHubTeacher adds Orlando, FL

- [x] create a branch
- [x] create some commits
- [x] open a pull request
- [x] collaborate + additional commits
- [ ] merge

### PR DESCRIPTION
**Pull request by JONGBURMPARK**
This PR closes #2.
This PR is for the purposes of adding the Sydney file to the hometown files.
Hey @JBPARK007 -- could you put some 👀 on this PR, please? Give me a 👍 if you think it's god to 🚢 . Thanks 🙇‍♂️ !

### CODE REVIEW

File: jongburmpark-sydney.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Sydney
+### Restaurants
+- Pedro pizza
+- Domino's
+
+### Tourist
```
Comment (JBPARK007) [inline jongburmpark-sydney.md:6]: @JONGBURMPARK Sydney is big for many tourists attractions.

**Comment by JONGBURMPARK** [comment]
@JBPARK007 -- Could you review this before 🚢  this?

File: jongburmpark-sydney.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Opera House
+- Harbour bridge
+
+### local attractions
```
Comment (JBPARK007) [inline jongburmpark-sydney.md:10]: This looks good :thus

File: jongburmpark-sydney.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- Epping Park
```
Comment (JBPARK007) [inline jongburmpark-sydney.md:12]: Epping park is not a local attraction.

**Review by JBPARK007 — APPROVED** [review]
@JONGBURMPARK I say :shipit:

**Pull request merged**